### PR TITLE
Kan 20 display available campaigns

### PR DESCRIPTION
**Pull request by matthew-estes**

### CODE REVIEW

File: src/components/CampaignsList/CampaignsList.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { Link } from "react-router-dom";
```
Comment (craigbwagner) [inline src/components/CampaignsList/CampaignsList.js:1]: Link isn't necessary here since you don't use it

**Review by craigbwagner — CHANGES_REQUESTED** [review]
go ahead and add the route to display this page in the App.jsx file

File: src/components/CampaignsList/CampaignsList.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+function CampaignList(props) {
+  return (
+    <div>
+      <h1>Available Campaigns</h1>
+      <ul>
+        {props.campaigns.map((campaign) => (
+          <li key={campaign._id}>
```
Comment (caojohnny91) [inline src/components/CampaignsList/CampaignsList.js:9]: everything looks good, but should we do a <Link key={campaign._id} to={`/campaigns/${campaign._id}`}> to make it a clickable link to its details page

**Review by caojohnny91 — APPROVED** [review]

File: src/components/CampaignsList/CampaignsList.js (hunk 1 above)
Comment (matthew-estes) [reply src/components/CampaignsList/CampaignsList.js]: ok

**Review by craigbwagner — APPROVED** [review]

**Pull request merged**